src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -1865,6 +1865,40 @@ JNI_METHOD(void, read)
     }
 }
 
+// Convert Json to Tlv, and remove the outer structure
+CHIP_ERROR ConvertJsonToTlvWithoutStruct(const std::string & json, MutableByteSpan & data, size_t & writtenLength)
```

Comment:
since mutablebytespan is already mutable, how about not returning writtenLength but instead calling `reduce_size` on it so that data becomes in/out (since this is what its purpose seems to be).

Reply:
Done, thanks